gravitee-apim-console-webui/src/management/api/apis-routing.module.ts

```diff
@@ -79,6 +79,8 @@ import { ApiPropertiesComponent } from './properties/properties/api-properties.c
 import { ApiNotificationComponent } from './api-notification/api-notification.component';
 import { ApiRuntimeAlertsComponent } from './runtime-alerts';
 import { ApiDocumentationV4MetadataComponent } from './documentation-v4/documentation-metadata/api-documentation-v4-metadata.component';
+import { ApiHistoryV4Component } from './history-v4/api-history-v4.component';
+import { DeploymentInfoComponent } from './history-v4/deployment-info/deployment-info.component';
```

Comment:
It is just a detail, but we often try to have consistent naming for component of module
no problem with the folder name, but I would have named the file `api-history-v4-deployment-info` (and rename the component in the same fashion)
@Okhelifi @ThibaudAV I would be curious to have your thoughts on this point

Reply:
sure, I changed it as you proposed to keep it aligned with the rest of the component's names

Reply:
I like it too, but it creates components with names that are too long. So you have to find a compromise between a clear name with "parent name" and "component name"
in that case it's not too long :p